src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -34,6 +34,8 @@ definitions:
         "$ref": "#/definitions/Jira"
       s3:
         "$ref": "#/definitions/s3"
+      camunda:
+        "$ref": "#/definitions/camunda"
```

Comment:
You also need to define the schema of the `camunda` object in the yaml. You can check how the s3 object is defined for example